src/twisted/conch/ssh/keys.py

```diff
@@ -256,24 +256,38 @@ def _fromString_BLOB(cls, blob):
         if keyType == b"ssh-rsa":
             e, n, rest = common.getMP(rest, 2)
             return cls(rsa.RSAPublicNumbers(e, n).public_key(default_backend()))
-        elif keyType == b"ssh-dss":
+
+        if keyType == b"ssh-dss":
             p, q, g, y, rest = common.getMP(rest, 4)
             return cls(
                 dsa.DSAPublicNumbers(
                     y=y, parameter_numbers=dsa.DSAParameterNumbers(p=p, q=q, g=g)
                 ).public_key(default_backend())
             )
-        elif keyType in _curveTable:
+
+        if keyType in _curveTable:
             return cls(
                 ec.EllipticCurvePublicKey.from_encoded_point(
                     _curveTable[keyType], common.getNS(rest, 2)[1]
                 )
             )
-        elif keyType == b"ssh-ed25519":
+
+        if keyType == b"sk-ecdsa-sha2-nistp256@openssh.com":
```

Comment:
I think that this change should be reverted, as this is out of scope for this PR.

the title of this PR is about updating the examples ... but here the main code is modified

Reply:
@adiroiban yep, wrongly I created this PR starting from your branch. What about the comment from @twm ? Does it make sense to have this kind of way to pass keys to the examples ?